lib/api/multiObjectDelete.js

```diff
@@ -124,7 +125,7 @@ function _formatXML(quietSetting, errorResults, deleted) {
         if (isDeletingDeleteMarker) {
```

Comment:
nvm, I looked at the code more, I guess deletingDeleteMarker is also true if we are creating a new delete marker?